thinkhazard/static/less/report.less

```diff
@@ -229,3 +229,25 @@ body {
     margin-bottom: 4px;
   }
 }
+
+.style-pdf-overview-btn {
+  padding: 7px 10px;
+  margin-top: 15px;
```

Comment:
The margin-top shouldn't not be set in the button itself. It actually means that the download button has to be 15px away from the navbar. But I wouldn't care much for now.